examples/tts/conf/magpietts/magpietts_dc_en_tiny.yaml

```diff
@@ -0,0 +1,175 @@
+name: Magpie-TTS-EN
```

Comment:
When do we use this yaml?

Reply:
This is the yaml for small version of magpietts